app/presenters/rejected_application_choice.rb

```diff
@@ -0,0 +1,127 @@
+class RejectedApplicationChoice < SimpleDelegator
```

Comment:
Minor style-police point: For better or for worse all our other presenters use `Presenter` in the class name. Not a blocker.